src/tests/ftest/launch.py

```diff
@@ -924,16 +1020,30 @@ def archive_config_files(avocado_logs_dir):
     # Copy any config files
     base_dir = get_build_environment()["PREFIX"]
     configs_dir = get_temporary_directory(base_dir)
-    archive_files(
-        destination, host_list, "{}/*_*_*.yaml".format(configs_dir))
+    task = archive_files(
+        destination, host_list, "{}/*_*_*.yaml".format(configs_dir), False,
+        args)
+
+    status = 0
+    if not check_remote_output(task, "archive_config_files"):
+        status = 16
+    return status
 
-def archive_files(destination, host_list, source_files):
+
+def archive_files(destination, hosts, source_files, cart, args):
     """Archive all of the remote files to the destination directory.
 
     Args:
         destination (str): path to which to archive files
-        host_list (list): hosts from which to archive files
+        hosts (list): hosts from which to archive files
         source_files (str): remote files to archive
+        cart (str): enable running cart_logtest.py
```

Comment:
```suggestion
        cart (bool): enable running cart_logtest.py
```
I think this argument is bool.

Reply:
Ah, good catch. I don't think it should warrant another run, but I'll make sure to fix it on follow up PR.